packages/cray-pre-install-toolkit/base.packages

```diff
@@ -15,6 +15,8 @@ metal-ipxe=2.2.3-1
 metal-net-scripts=0.0.2-1
 pit-init=1.2.15-1
 pit-nexus=1.1.0-1.1
+tpm2.0-abrmd=2.3.3-1.12
```

Comment:
Need to remove move this, moving it into `metal.packages` instead.

Reply:
This one if for the PIT.  `metal.packages` is set in here: https://github.com/Cray-HPE/csm-rpms/pull/313/files

Reply:
Then this just needs to be removed. I said "moved" before because it shouldn't be in base.packages, period.

Reply:
fixed